app/assets/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.js

```diff
@@ -0,0 +1,32 @@
+;(function (global) {
+  'use strict'
+
+  var GOVUK = global.GOVUK || {}
+  GOVUK.Modules = GOVUK.Modules || {}
+
+  GOVUK.Modules.ExplicitCrossDomainLinks = function () {
+    this.start = function ($module) {
+      this.$module = $module[0]
+
+      if (!global.ga) { return }
+
+      var trackers = global.ga.getAll()
+
+      if (!trackers.length) { return }
+
+      var linker = new global.gaplugins.Linker(trackers[0])
+
+      var attrAction = this.$module.getAttribute('action')
+      if (attrAction) {
+        this.$module.setAttribute('action', linker.decorate(attrAction))
+      }
+
+      var attrHref = this.$module.getAttribute('href')
+      if (attrHref) {
+        this.$module.setAttribute('href', linker.decorate(attrHref))
```

Comment:
I'm _pretty sure_ you don't need `setAttribute` for this. I could be totally wrong so if you test this and it doesn't work or you've already tried this then please ignore. If it does work you might be able to do the same for the `action` attribute above but I'm not sure how this would play with older browsers.

```suggestion
        this.$module.href = linker.decorate(attrHref)
```

Reply:
Great, it works for `href`.  How could I make sure `this.$module.action` is fine for older browsers?

Reply:
The best way to do this is to just check on older browsers. If you don't have access to browserstack you can get it by going via Matt Hobbs _or_ I could test this. Having said that, if you're under pressure to get this out, I'm not precious about this so we can ignore it.